dist/tools/testrunner/testrunner.py

```diff
@@ -37,7 +37,8 @@ def find_exc_origin(exc_info):
 
 def run(testfunc, timeout=10, echo=True, traceback=False):
     env = os.environ.copy()
-    child = pexpect.spawnu("make term", env=env, timeout=timeout, codec_errors='replace')
+    child = pexpect.spawnu("make term", env=env, timeout=timeout,
```

Comment:
This isn't compliant with PEP-8: https://www.python.org/dev/peps/pep-0008/#id17.

It should be aligned like this:

```python
child = pexpect.spawnu(
    "make term", env=env, timeout=timeout, codec_errors='replace')
```